lms/templates/discovery/more_less_links.underscore

```diff
@@ -0,0 +1,8 @@
+<div class="toggle ">
+    <a aria-hidden="true" tabindex="-1" class="show-more" href="#">
```

Comment:
What does this do?  Does the aria-hidden ever get removed?  Why don't you want it to be accessible?  Focus will not be able to be put on this element for a screen reader user if tabindex=-1

Should be a ```<button>```   Same for LESS below